src/NumericalAlgorithms/Spectral/LogicalCoordinates.cpp

```diff
@@ -22,11 +23,63 @@ void logical_coordinates(
     const Mesh<VolumeDim>& mesh) {
   set_number_of_grid_points(logical_coords, mesh.number_of_grid_points());
   for (size_t d = 0; d < VolumeDim; ++d) {
-    const auto& collocation_points_in_this_dim =
-        Spectral::collocation_points(mesh.slice_through(d));
-    for (IndexIterator<VolumeDim> index(mesh.extents()); index; ++index) {
-      logical_coords->get(d)[index.collapsed_index()] =
-          collocation_points_in_this_dim[index()[d]];
+    switch (mesh.basis(d)) {
+      case Spectral::Basis::SphericalHarmonic: {
+        switch (mesh.quadrature(d)) {
+          case Spectral::Quadrature::Gauss: {
```

Comment:
The basis functions are not defined on -1 to 1. It's only the standard Jacobi polynomials that are defined on this interval. There are basis functions defined on `0, 1`, `0, infty`, `-infty, 0`, `-infty, infty`, `0, pi`, etc. It's just that Jacobi polynomials happen to be orthonormal on `[-1, 1]`, that's all. There's nothing generally special about those two numbers. This is just how basis functions work. I have nothing against stating the domain of the basis functions (we should do so for all of them, then!), but we shouldn't add any language implying there's something "weird" because they aren't on `-1, 1` and that we magically "hid" the `-1, 1` coordinates. That's not what's happening, there are no `-1, 1` coordinates for all basis functions.

Reply:
Sounds good, there is a relation to Jacobi polynomials though since we use Legendre-Gauss points in cos(theta). Anyway, just stating the domain of the logical coords seems useful, and stating what collocation points are used for the ylm basis.

Reply:
I expanded the documentation.